hudi-cli/src/main/java/org/apache/hudi/cli/commands/HDFSParquetImportCommand.java

```diff
@@ -78,8 +76,8 @@ public String convert(
       cmd = SparkCommand.UPSERT.toString();
     }
 
-    sparkLauncher.addAppArgs(cmd, srcPath, targetPath, tableName, tableType, rowKeyField, partitionPathField,
-        parallelism, schemaFilePath, sparkMemory, retry, propsFilePath);
+    sparkLauncher.addAppArgs(cmd, master, sparkMemory, srcPath, targetPath, tableName, tableType, rowKeyField,
```

Comment:
As a refactor suggestion, it would be better to define a data structure to store the cli args to avoid change the signature of the method frequently.